scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -109,14 +109,28 @@ DisableStrongNameVerification -publicKeyToken '31bf3856ad364e35' # NuGet
 $regKeyFileSystem = "HKLM:SYSTEM\CurrentControlSet\Control\FileSystem"
 $enableLongPathSupport = "LongPathsEnabled"
 
-# update submodule NuGet.Build.Localization
 $NuGetClientRoot = $env:BUILD_REPOSITORY_LOCALPATH
 $Submodules = Join-Path $NuGetClientRoot submodules -Resolve
 
+# NuGet.Build.Localization repository set-up
 $NuGetLocalization = Join-Path $Submodules NuGet.Build.Localization -Resolve
-$NuGetLocalizationRepoBranch = 'master'
-$updateOpts = 'pull', 'origin', $NuGetLocalizationRepoBranch
 
+# Check if there is a localization branch associated with this branch repo 
+$currentNuGetBranch = $env:BUILD_SOURCEBRANCHNAME
+$lsRemoteOpts = 'ls-remote', 'origin', $currentNuGetBranch
+Write-Host "Looking for branch '$currentNuGetBranch' in NuGet.Build.Localization"
+$lsResult = & git -C $NuGetLocalization $lsRemoteOpts
+
+if ($lsResult) {
+    $NuGetLocalizationRepoBranch = $currentNuGetBranch
+}
+else {
+    $NuGetLocalizationRepoBranch = 'dev'
+}
+Write-Host "NuGet.Build.Localization Branch: $NuGetLocalizationRepoBranch"
```

Comment:
I really like logging this info. However, looking at the build, I don't see the message in the log. Did I look in the wrong place, or is it really missing?

Reply:
never mind, I was looking at the wrong log! 😞